packages/website/package.json

```diff
@@ -13,11 +13,11 @@
   },
   "dependencies": {
     "@algolia/ui-library": "2.35.0",
-    "@docsearch/react": "1.0.0-alpha.28",
+    "@docsearch/react": "3.0.0-alpha.36",
```

Comment:
This website package isn't the live https://docsearch.algolia.com/ one, I've updated the dependencies just to make it consistent.